test/scaler/scaler_native_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Scaler.NativeTest do
+  use ExUnit.Case, async: true
+  alias Membrane.Payload
+  alias Membrane.FFmpeg.SWScale.Scaler.Native, as: Scaler
+
+  test "Scale 1 720p frame" do
+    in_path = "../fixtures/input-10-1280x720.raw" |> Path.expand(__DIR__)
+
+    assert {:ok, file} = File.read(in_path)
+    assert {:ok, native_state} = Scaler.create(1280, 720, 640, 640)
```

Comment:
This code is not fully tested - make sure to check scenarios where
- input is vertical
- input is a square
- output is vertical

in different combinations